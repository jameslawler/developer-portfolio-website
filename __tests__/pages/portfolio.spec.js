import React from 'react';
import renderer from 'react-test-renderer';

import Portfolio from '../../pages/portfolio';

describe('With Snapshot Testing', () => {
  it('Portfolio component should render', () => {
    const result = renderer.create(<Portfolio />).toJSON();
    expect(result).toMatchSnapshot();
  });
});
