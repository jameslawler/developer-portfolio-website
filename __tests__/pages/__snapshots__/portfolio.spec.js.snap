// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Portfolio component should render 1`] = `
<div>
  <p>
    Portfolio Markdown:
  </p>
  <p>
    Portfolio Data:
  </p>
  <div />
</div>
`;
